### PR TITLE
Localize 'email address' placeholder

### PR DESCRIPTION
**Pull request by Arhell**
[issue](https://github.com/kubernetes/website/issues/19386)
/language ru
/assign @php-coder

### CODE REVIEW

**Comment by Arhell** [comment]
[en issue](https://github.com/kubernetes/website/issues/19272)

File: i18n/ru.toml [hunk 1]
```diff
@@ -193,3 +193,6 @@ other = "Календарь событий"
 # UI elements
 [ui_search_placeholder]
 other = "Поиск"
+
+[input_placeholder_email_address]
+other = "адрес электронной почты"
```
Comment (php-coder) [inline i18n/ru.toml]: Do we need to append a newline character here?

**Comment by php-coder** [comment]
@Arhell Hi, thank you for the contribution!

The translation looks good but just to ensure I'd like to check rendered form. Do you happen to know where (on what page(s) I can see the result?

**Comment by Arhell** [comment]
@php-coder 
[issue](https://github.com/kubernetes/website/issues/19272)
En example 
https://github.com/kubernetes/website/pull/19359 was merge
and link home page https://kubernetes.io/
at the bottom form

**Comment by php-coder** [comment]
Thanks!
One more
/lgtm
from myself.

**Comment by sftim** [comment]
@php-coder are you happy to /approve (for the record)?

**Comment by php-coder** [comment]
@sftim Yes, this can be merged.

**Comment by sftim** [comment]
@php-coder you can tell Prow, by replying “/approve”

**Comment by sftim** [comment]
(what I'd expect is one of the Russian localization approvers to /approve, and then the English localization approvers can echo that to make Prow happy)

**Comment by php-coder** [comment]
> @php-coder you can tell Prow, by replying “/approve”

I can try but I doubt that it will work as I don't have approval rights.

/approve

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-19393--kubernetes-io-master-staging.netlify.com/ru/

**Comment by kbhawkey** [comment]
/lgtm
Ping @kubernetes/sig-docs-ru-owners

**Comment by sftim** [comment]
echoing https://github.com/kubernetes/website/pull/19393#pullrequestreview-368578382
/approve

**Pull request merged**